dvc/config_schema.py

```diff
@@ -225,5 +225,5 @@ class RelPath(str):
         # enabled by default. It's of no use, kept for backward compatibility.
         Optional("parametrization", default=True): Bool
     },
-    "plots": {"html_template": str},
+    "plots": {"html": str},
```

Comment:
Can it compatible with the old schema? Some users might have an old version in their workspace.

Reply:
This is true, but the feature is relatively new and I think there is more harm in leaving `html_template` because users might associate `template` with vega templates and not understand what it actually is.

Reply:
Can we adopt the old version, and transfer it to new ones? Or at least give some friendly messages when it happens.

Reply:
This feature was only introduced in the last minor version or so, and it was specifically developed for the Studio team, so I wouldn't expect many (probably any) other people are using it. Feel free to support both or whatever you think is best, but I don't think this is a breaking change worth worrying about.

Reply:
AFAIK @mnrozhkov is the person that needs to know about this change, also one user on discord. I will try to reach out to this user.

Reply:
Studio guys will complain about problems with this, we should not break an already released schema. We __have__ to keep older name, but we could introduce an alias.

Reply:
My mistake, I didn't think it would be a big deal for studio.

Reply:
Related review, for the record: https://github.com/iterative/dvc/pull/5694#pullrequestreview-690401273.